extra_data/write_cxi.py

```diff
@@ -21,15 +24,14 @@ class VirtualCXIWriter:
     Parameters
     ----------
 
-    detdata: extra_data.components.MPxDetectorBase
+    detdata: extra_data.components.XtdfDetectorBase
```

Comment:
We are back to this base class taking a `MPxDetectorBase` again now, I think.

Reply:
Yes, changed in https://github.com/European-XFEL/EXtra-data/commit/a8c2248ec48f2cbf6a4e7d79afc0b6981c49cb98 :)